### PR TITLE
doc: Update open trainings

### PR DESCRIPTION
**Pull request by The-Compiler**
The training is sold out with 10 people (yay!), so there's a second date for it now.

Is there a better way we can keep pytest.org up to date with those changes? From what I understand, it only gets updated when we make a new release from `main`, which hasn't happened this year so far? It's kinda troublesome we still advertise a training from February (with "Early-bird discount available until January 15th.") there... 

I struggle to think of a better solution though... Can we (and do we want to) reconfgure RTD so it shows "latest" by default instead, or would that be too confusing if we had breaking changes?

### CODE REVIEW

**Review by bluetech — APPROVED** [review]

**Comment by bluetech** [comment]
> I struggle to think of a better solution though... Can we (and do we want to) reconfgure RTD so it shows "latest" by default instead, or would that be too confusing if we had breaking changes?

It used to be pointed at latest but we changed it to stable for this reason.

I don't have any bright ideas besides cherry-picking these over to the stable branch (can merge without waiting for approval) and point the RTD `stable` at the stable *branch* (rather than release) if that's even possible.

**Comment by The-Compiler** [comment]
I can take a look but I assume someone would need to give me access? I just created an account over there, `The-Compiler`.

**Comment by nicoddemus** [comment]
> I just created an account over there, The-Compiler.

Done! 👍

**Comment by The-Compiler** [comment]
Taking the freedom to merge this as the failures are unrelated. Will follow up with a backport too.

**Pull request merged**
